index.html

```diff
@@ -122,6 +122,33 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Paul Kim</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com" target="_blank">dsk15_</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">paulkim430</a>
```

Comment:
Please note that you need to insert the links to your profiles and not to platforms ( eg. just Twitter & GitHub ). Kindly update that.